src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;

```

Comment:
All magic numbers in your code should be constants. You could define PI as a constant. Although `Math.PI` is a constant itself, if you had other magic numbers, they should be constants.